src/SprykerEco/Zed/AfterPay/Communication/Plugin/Checkout/AfterPayPreCheckPlugin.php

```diff
@@ -33,11 +34,6 @@ class AfterPayPreCheckPlugin extends AbstractPlugin implements CheckoutPreSaveHo
      */
     public function preSave(QuoteTransfer $quoteTransfer, CheckoutResponseTransfer $checkoutResponseTransfer): QuoteTransfer
     {
-        $afterPayCallTransfer = $this->getFactory()
-            ->createQuoteToCallConverter()
-            ->convert($quoteTransfer);
-        $this->getFacade()->authorizePayment($afterPayCallTransfer);
-
         return $quoteTransfer;
```

Comment:
did you move the logic from AfterPayPreCheckPlugin to AfterPaySaveOrderPlugin ?

It changes the plugin logic completely, so this is plugin is not needed after this change because does nothing. It means you can revert the change and deprecated this plugin in favor of `AfterPaySaveOrderPlugin`. `@deprecated Will be removed without replacement. Use {@link SprykerEco\Zed\AfterPay\Communication\Plugin\Checkout\AfterPaySaveOrderPlugin`} instead.

Reply:
FIxed